carddrawer/src/main/res/layout/card_view_generic_image_container.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.meli.android.carddrawer.model.ConstraintLayoutWithDisabledSupport xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:showIn="@layout/card_drawer_generic_front">
+
+    <androidx.cardview.widget.CardView
```

Comment:
Do we know why it works this way? Since before it was inside a ConstraintLayoutWithDisabledSupport

Reply:
In this case, the ConstraintLayoutWithDisabledSupport is not needed, since the parent has it. With a ContrainstLayout we are ok. In the future we can make a kind of recursion in the drawChild method, to avoid these cases :P

Reply:
This change I am going to make in the card view pr